include/deal.II/fe/fe_interface_values.h

```diff
@@ -2496,11 +2496,22 @@ FEInterfaceValues<dim, spacedim>::reinit(
           {cell->active_fe_index(), cell_neighbor->active_fe_index()});
 
       const unsigned int used_q_index =
-        (q_index == numbers::invalid_unsigned_int ? dominated_fe_index :
-                                                    q_index);
+        [q_index, dominated_fe_index, &cell]() -> unsigned int {
+        if (q_index != numbers::invalid_unsigned_int)
+          return q_index;
+        if (dominated_fe_index != numbers::invalid_fe_index)
+          return dominated_fe_index;
+        return cell->active_fe_index();
+      }();
```

Comment:
As stated elsewhere, I don't think this is right. We should be choosing the same quadrature formula regardless of which side of a face you approach.